deployment/buildAndDeploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ $TRAVIS_BRANCH == 'master' ]]; then
+  ./gradlew uploadArchives -PnexusUser=$NEXUS_USER -PnexusPassword=$NEXUS_PASSWORD -Psigning.keyId=$SIGNING_KEY_ID -Psigning.password=$SIGNING_PASSWORD -Psigning.secretKeyRingFile=../deployment/secret-keys-keanu.gpg
```

Comment:
It looks like `uploadArchives` depends on `compileJava` but it doesn't depend on `check`, which we should still be running on master.

I think the best command to use is `./gradlew clean build uploadArchives`

Also worth sticking a `--info --stacktrace` on the end too.

Reply:
Nice I'll add that in